samplers/samplers.go

```diff
@@ -324,35 +336,40 @@ func (s *Set) Combine(other []byte) error {
 	return nil
 }
 
+// GetName returns the Name of the Set
+func (s *Set) GetName() string {
+	return s.Name
+}
+
 // Histo is a collection of values that generates max, min, count, and
 // percentiles over time.
 type Histo struct {
-	Name  string
-	Tags  []string
-	Value *tdigest.MergingDigest
+	Name    string
+	Tags    []string
+	tDigest *tdigest.MergingDigest
 	// these values are computed from only the samples that came through this
 	// veneur instance, ignoring any histograms merged from elsewhere
 	// we separate them because they're easy to aggregate on the backend without
 	// loss of granularity, and having host-local information on them might be
 	// useful
```

Comment:
looks like you renamed the variables to reflect the fact that they're not always local-only, comment should probably be updated as well?